app/models/appropriate_body.rb

```diff
@@ -1,5 +1,14 @@
 class AppropriateBody < ApplicationRecord
-  NAME_FOR_TEACHING_INDUCTION_PANEL_TYPE = 'Independent Schools Teacher Induction Panel (ISTIP)'.freeze
+  ISTIP = 'Independent Schools Teacher Induction Panel (ISTIP)'.freeze
+
+  AppropriateBody.inheritance_column = nil
+
+  # Enums
+  enum :type,
+       { local_authority: 'local_authority',
+         national: 'national',
+         teaching_school_hub: 'teaching_school_hub' },
+       validate: { message: "Must be local authority, national or teaching school hub" }
```

Comment:
Could we use something other than `type` for the name e.g. `ab_type` or `body_type` instead just so we don't cause confusion and stick to the Rails conventions?

Reply:
Probably worth getting input on this from @emilyprudencedfe or @priyanahar2

Reply:
No problem to rename it but Rails provides `inheritance_column` precisely for this